spec/models/atmosphere/user_fund_spec.rb

```diff
@@ -33,4 +33,75 @@
         to include(I18n.t('funds.unique_user'))
   end
 
+
+  describe '#ensure_single_default' do
+
+    it 'makes first user_fund default' do
+      expect(user_fund.default).to be_truthy
+    end
+
+    it 'blocks setting the only fund as not default' do
+      expect(user_fund.default).to be_truthy
+      user_fund.default = false
+      user_fund.save
+      expect(user_fund.reload.default).to be_truthy
```

Comment:
There will be no errors since this is not regarding validation. Without reload most of those tests gave me false positives or false negatives. So I decided to be on the safe side so I reload them.